### PR TITLE
Implement teachinginternship.yml as variables

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/2WHavuTv

### Context

- We are centralising our monetary values into one place, this PR implements this.

### Guidance to review

Pages where this is changing:

[ ] https://get-into-teaching-app-review-4028.test.teacherservices.cloud/is-teaching-right-for-me
[ ] https://get-into-teaching-app-review-4028.test.teacherservices.cloud/is-teaching-right-for-me/teaching-internship-providers

### Pre-election period restrictions

* [ ] This change is permitted within the constraints of the [pre-election period guidelines](https://www.gov.uk/government/publications/election-guidance-for-civil-servants/general-election-guidance-2024-guidance-for-civil-servants-html#publishing-content-online-)

### CODE REVIEW

File: lib/tasks/support/teaching-internship-providers.md.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@ backlink: /
 fullwidth: true
 navigation: 5.55
 navigation_title: Can I do a teaching internship?
-navigation_description: Earn from £300 per week and find out what it is really like in the classroom with a teaching internship.
+navigation_description: Earn from <%= v :teachinginternship_weekly %> per week and find out what it is really like in the classroom with a teaching internship.
```
Comment (gemmadallmandfe) [inline lib/tasks/support/teaching-internship-providers.md.erb:15]: @MylesJarvis You've probably seen that this doesn't render properly on `/is-teaching-right-for-me`. Let's discuss before looking for a dev solution!

File: app/views/content/is-teaching-right-for-me/teaching-internship-providers/_listing.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <p>If you’re studying for an undergraduate or master's degree and interested in teaching, an internship could help you to understand what it’s really like in the classroom.</p>
 
-<p>Internships last 3 weeks, start in June or July and you’ll be paid from £300 per week.</p>
+<p>Internships last 3 weeks, start in June or July and you’ll be paid from <%= v :teachinginternship_weekly %>% per week.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/is-teaching-right-for-me/teaching-internship-providers/_listing.html.erb:3]:
```suggestion
<p>Internships last 3 weeks, start in June or July and you’ll be paid from <%= v :teachinginternship_weekly %> per week.</p>
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by Sarah-DfE — COMMENTED** [review]
Looks good to me too @gemmadallmandfe

**Pull request merged**